extensions/ql-vscode/test/unit-tests/databases/config/db-config-store.test.ts

```diff
@@ -496,4 +601,178 @@ describe("db config store", () => {
+        remoteLists: [
+          {
+            name: "list1",
+            repositories: ["owner/repo1", "owner/repo2"],
+          },
+        ],
+      });
+
+      const configStore = await initializeConfig(dbConfig, configPath, app);
+
+      // Check
+      const doesExist = await configStore.doesRemoteListExist("list1");
+      expect(doesExist).toEqual(true);
+
+      configStore.dispose();
+    });
+
+    it("should return true if a remote db exists", async () => {
+      // Initial set up
+      const dbConfig = createDbConfig({
+        remoteLists: [
+          {
+            name: "list1",
+            repositories: ["owner/repo1", "owner/repo2"],
+          },
+        ],
+      });
+
+      const configStore = await initializeConfig(dbConfig, configPath, app);
+
+      // Check
+      const doesExist = await configStore.doesRemoteDbExist(
+        "owner/repo1",
+        "list1",
+      );
+      expect(doesExist).toEqual(true);
+
+      configStore.dispose();
+    });
+
+    it("should return true if a local db and local list exists", async () => {
+      // Initial set up
+      const dbConfig = createDbConfig({
+        localLists: [
+          {
+            name: "list1",
+            databases: [createLocalDbConfigItem({ name: "db1" })],
+          },
+        ],
+      });
+
+      const configStore = await initializeConfig(dbConfig, configPath, app);
+
+      // Check
+      const doesDbExist = await configStore.doesLocalDbExist("db1", "list1");
+      expect(doesDbExist).toEqual(true);
+      const doesListExist = await configStore.doesLocalListExist("list1");
+      expect(doesListExist).toEqual(true);
+
+      configStore.dispose();
+    });
+
+    it("should return false if items do not exist", async () => {
+      // Initial set up
+      const dbConfig = createDbConfig({});
+
+      const configStore = await initializeConfig(dbConfig, configPath, app);
+
+      // Check
+      const doesLocalDbExist = await configStore.doesLocalDbExist(
+        "db1",
+        "list1",
+      );
+      expect(doesLocalDbExist).toEqual(false);
+      const doesLocalListExist = await configStore.doesLocalListExist("list1");
+      expect(doesLocalListExist).toEqual(false);
+      const doesRemoteDbExist = await configStore.doesRemoteDbExist(
+        "db1",
+        "list1",
+      );
+      expect(doesRemoteDbExist).toEqual(false);
+      const doesRemoteListExist = await configStore.doesRemoteListExist(
+        "list1",
+      );
+      expect(doesRemoteListExist).toEqual(false);
+      const doesRemoteOwnerExist = await configStore.doesRemoteOwnerExist(
+        "owner1",
+      );
+      expect(doesRemoteOwnerExist).toEqual(false);
+
+      configStore.dispose();
+    });
+  });
+
+  async function initializeConfig(
+    dbConfig: DbConfig,
+    configPath: string,
+    app: App,
+  ): Promise<DbConfigStore> {
+    await writeJSON(configPath, dbConfig);
+
+    const configStore = new DbConfigStore(app);
```

Comment:
```suggestion
    const configStore = new DbConfigStore(app, false);
```

There's now an option to initialize the config store without starting the file watcher, which is what we generally want in tests. Let's do that here too!

Reply:
👍